mountpoint-s3-client/Cargo.toml

```diff
@@ -11,6 +11,7 @@ description = "High-performance Amazon S3 client for Mountpoint for Amazon S3."
 mountpoint-s3-crt = { path = "../mountpoint-s3-crt", version = "0.7.0" }
 mountpoint-s3-crt-sys = { path = "../mountpoint-s3-crt-sys", version = "0.7.0" }
 
+async-channel = "2.1.1"
```

Comment:
Is the channel provided by tokio (which we already have a dep on) useful here?

https://docs.rs/tokio/latest/tokio/sync/mpsc/index.html